group_vars/vars

```diff
@@ -0,0 +1,22 @@
+# Variables for ansible provisioning
+# See https://docs.ansible.com/ansible/latest/playbooks_variables.html
+
+# Paths (from scripts/_path_resolve.sh)
+ROOTDIR: "{{ rbenv_root }}"
+CONFDIR: "{{ ROOTDIR }}/configs"
+MWCONFFILE: "{{ ROOTDIR }}/mediawiki.php"
+SCRIPTDIR: "{{ ROOTDIR }}/scripts"
+PAGESDIR: "{{ SCRIPTDIR }}/pages"
+DUMPSDIR: "{{ ROOTDIR }}/dumps"
+WIKIFOLDER: wiki
+WIKIDIR: "{{ ROOTDIR }}/public/$WIKIFOLDER"
+VAGRANT_ROOTDIR: /var/www
+VAGRANT_CONFDIR: "{{ VAGRANT_ROOTDIR }}/configs"
+VAGRANT_MWCONFFILE: "{{ VAGRANT_CONFDIR }}/mediawiki.php"
+VAGRANT_SCRIPTDIR: "{{ VAGRANT_ROOTDIR }}/scripts"
+VAGRANT_PAGESDIR: "{{ VAGRANT_SCRIPTDIR }}/pages"
+VAGRANT_DUMPSDIR: "{{ VAGRANT_ROOTDIR }}/dumps"
+VAGRANT_WIKIDIR: "{{ VAGRANT_ROOTDIR }}/public/wiki"
+
+# Hitchwiki
+hw_owners: "{{ HW__general__webserver_user }}:{{ HW__general__webserver_group }}"
```

Comment:
Note that `HW__general__webserver_user` [were set with bash](https://github.com/Hitchwiki/hitchwiki/blob/master/scripts/_settings.sh) from `settings.ini`, but here they might need to be set just as strings? They could be taken out form settings.ini as this doesn't really need to be configurable. `www-data:www-data` is the default Apache2 user:group.

Reply:
TODO as soon as i figure out, how to include them from a script, they will depend on _settings.sh, or if we decide to deprecate it, from settings.ini.

Reply:
I think there's a bunch of scripts now relying on `_settings.ini`, although those can probably all be migrated to Ansible.

`settings.sh` setting up env vars for Ansible could be [one trick](https://docs.ansible.com/ansible/latest/intro_configuration.html#getting-the-latest-configuration), although in general I always feel like they are a bit of a blackbox so I try to avoid them.